src/Features/CSharp/Portable/OrderModifiers/CSharpOrderModifiersCodeFixProvider.cs

```diff
@@ -16,5 +17,9 @@ public CSharpOrderModifiersCodeFixProvider()
             : base(CSharpSyntaxFactsService.Instance, CSharpCodeStyleOptions.PreferredModifierOrder, CSharpOrderModifiersHelper.Instance)
         {
         }
+
+        private const string CS0267 = nameof(CS0267); // The 'partial' modifier can only appear immediately before 'class', 'struct', 'interface', or 'void'
```

Comment:
usually we put const at the top of the class?

Reply:
Thanks, I didn't know that. I will update this PR later.

Reply:
Fixed in 2366fbc860201049378d0843ff48632fdceb8bfa